cdap-ui/app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -0,0 +1,211 @@
+          />
+        </Col>
+      </FormGroup>
+    );
+  };
+
+  renderDescription = () => {
+    return (
+      <FormGroup row>
+        <Col xs="3">
+          <strong
+            className="label"
+            id="profile-description"
+          >
+            Description
+          </strong>
+        </Col>
+        <Col xs="5">
+          <Input
+            type="textarea"
+            aria-labelledby="profile-description"
+            value={this.state.profileDescription}
+            onChange={this.onMetadataChange.bind(this, 'profileDescription')}
+            placeholder="Add a description for the profile"
+          />
+        </Col>
+      </FormGroup>
+    );
+  };
+
+  renderGroup = (group) => {
+    return (
+      <div className="group-container" key={group.name}>
+        <strong className="group-title"> {group.name} </strong>
+        <hr />
+        <div className="group-description">
+          {group.description}
+        </div>
+        <div className="fields-container">
+          {
+            group.properties.map(property => {
+              let uniqueId = `${group.name}-${property.name}`;
+              return (
+                <FormGroup key={uniqueId} row>
+                  <Col xs="3">
+                    <strong
+                      className="label"
+                      id={uniqueId}
+                    >
+                      {property.label}
+                    </strong>
+                  </Col>
+                  <Col xs="5">
+                    {
+                      <AbstractWidget
+                        type={property['widget-type']}
+                        value={""}
+                        onChange={this.onValueChange.bind(this, property.name)}
+                        widgetProps={property['widget-attributes']}
+                      />
+                    }
+                  </Col>
+                </FormGroup>
+              );
+            })
+          }
+        </div>
+      </div>
+    );
+  };
+
+  render() {
+    let configurationGroups = SampleViewSpecJson['configuration-groups'];
+    return (
+      <div className="profile-create-view">
+        <div className="create-view-top-panel">
+          Create a Goolge Dataproc Profile
+        </div>
+        <div className="create-form-container">
+          <Form
+            className="form-horizontal"
+            onSubmit={(e) => {
+              preventPropagation(e);
+              return false;
+            }}
+          >
+            {this.renderProfileName()}
+            {this.renderDescription()}
+            {
+              configurationGroups.map(group => this.renderGroup(group))
+            }
+          </Form>
+        </div>
+        <div className="btns-section">
+          <Button
+            color="primary"
+            onClick={this.createProfile}
+            disabled={!this.state.profileName.length && !this.state.profileDescription.length}
```

Comment:
I think this should be `||` instead of `&&`. Right now if I fill in the description but not the name, the button is still enabled.